libs/plugin-types/index.d.ts

```diff
@@ -487,7 +505,7 @@ export interface PenpotFrameGuideSquare {
   /**
    * The type of the guide, which is always 'column' for square guides.
    */
-  type: 'column';
+  type: 'square';
```

Comment:
Not sure if it is a mistake, but the comment above says its always `column`

Reply:
The type was wrong, i forgot to update the comment as well.